src/index.css

```diff
@@ -38,6 +38,9 @@
   --selection: #ff7bed35;
   --transparent: #00000000;
 
+  --accent-transparent: rgba(254, 216, 35, 0.1);
```

Comment:
Maybe we can use `hex` here? You can specify hex alpha channel like I did in `--transparent`

Reply:
Sure, but I thought it's easy to read the color if it set a rgba. But yeah, why not